src/settings.js

```diff
@@ -1341,6 +1341,8 @@ var DYNCALLS = false;
 // WebAssembly integration with JavaScript BigInt. When enabled we don't need
 // to legalize i64s into pairs of i32s, as the wasm VM will use a BigInt where
 // an i64 is used.
+// If WASM_BIGINT is present, the default minimum browser versions increase to
+// Safari v15.0, Firefox v68, and Edge v79.
```

Comment:
What about Chrome?

Reply:
Chrome is unchanged because the default `MIN_CHROME_VERSION` supports `BigInt` and `BigInt64Array`. Maybe I should say that explicitly.